### ISSUE TITLE
Lights are down between unread chapters

### ISSUE THREAD

**Issue by arfannn88** [tachiyomiorg/tachiyomi#9121]
### Steps to reproduce

Lights are still down between unread chapters. When we read a chapters the light went down for the chapters but but now some chapters are already light down even before we started reading. We users thought it will be fixed by next update but it didn't fix.

### Expected behavior

The light shouldn't down before we even started reading it. 

### Actual behavior

Lights are still down between unread chapters. When we read a chapters the light went down for the chapters but but now some chapters are already light down even before we started reading. We users thought it will be fixed by next update but it didn't fix.

### Tachiyomi version

0.14.5

### Android version

Android 11

### Device

All android devices 

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have gone through the [FAQ](https://tachiyomi.org/help/faq/) and [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to version **[0.14.5](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by arfannn88** [comment]
![Screenshot_2023_0221_220341](https://user-images.githubusercontent.com/124960383/220397091-bcc8b7af-0fe0-4e61-9aab-f0cf1c699d8b.png)
